src/components/attendee/SearchOverlay/SearchOverlay.tsx

```diff
@@ -77,6 +97,17 @@ export const SearchOverlay: React.FC<SearchOverlayProps> = ({ onClose }) => {
     );
   }, [searchQuery, enabledRelatedPortals]);
 
+  const foundSpaces = useMemo<SpaceWithId[]>(() => {
+    if (!searchQuery) return ALWAYS_EMPTY_ARRAY;
+
+    /* @debt we really shouldn't be using the index as part of the key here, it's unstable.. but portals don't have a unique identifier */
```

Comment:
is this comment relevant?.. looks like it's not.

Reply:
[09146a2](https://github.com/sparkletown/sparkle/pull/3074/commits/09146a2a8a1fd2ccbb351fd1056f0487e6181767)